src/hubverse_transform/model_output.py

```diff
@@ -26,14 +26,23 @@ def __init__(self, input_uri: str, output_uri: str):
         self.fs_output = output_filesystem[0]
         self.output_path = output_filesystem[1]
 
+        self.valid_file = True
+
```

Comment:
is this used anywhere?

Reply:
I am not, and am surprised the linter didn't yell about this.